src/webapp/components/data-submission/DataSubmissionContent.tsx

```diff
@@ -33,19 +36,36 @@ export const DataSubmissionContent: React.FC = () => {
                         changeStep={changeStep}
                     />
                     {stepsResult?.data[0]?.children?.length &&
-                        (currentStep === 3 ? (
-                            <>
-                                <ConsistencyChecks />
-                                <SupportButtons />
-                            </>
-                        ) : (
-                            <p className="intro">{i18n.t(stepsResult.data[0].children[currentStep - 1]?.content)}</p>
-                        ))}
+                        renderStep(
+                            currentStep,
+                            changeStep,
+                            i18n.t(stepsResult.data[0].children[currentStep - 1]?.content)
```

Comment:
That's tricky, this string won't be translated. i18n infrastructure (gettext) requires `i18n.t("Something")`, so it can be extracted. So the content fields must do the `i18n.t`, not here.

Reply:
follow-up on this task: https://app.clickup.com/t/860pp6vuu